src/lcd.c

```diff
@@ -522,11 +522,11 @@ void LCD_PrintHexW(uint16_t value, uint8_t mode)
  *  \note  You have to call \ref LCD_Update() to trigger update on LCD if not
  *         it is triggered automatically at change of bitframe
  *
- *  \note  range for desired temperature 5,0�C - 30�C, OFF and ON
+ *  \note  range for desired temperature 5,0�C - 30�C, OFF and ON
  *
```

Comment:
This is probably unwanted change, right? Having special character like the degree sign can pose problem, but changing it to nonsense character won't help :-). The same issue below, on more places.

Reply:
oops, should have checked, that every file is utf-8 encoded.. will change this!